### ISSUE TITLE
Incorrect displays size

### ISSUE THREAD

**Issue by Roman-Dzerkal** [leanflutter/screen_retriever#11]
Hello! There are 2 displays. Each has resolution = 1920x1080.
Display **1** is **primary**.
![изображение](https://user-images.githubusercontent.com/84628556/217022575-276bb67f-eab2-4820-92e8-7bf421cbf616.png)

I have next code, which shows info about displays in different positions:

```
  List<Display> displaysList = await screenRetriever.getAllDisplays();
  displaysList.forEach((display) {
    print(display.toJson());
  });
```
I will show only necessary info.
> name: \\.\DISPLAY1, size: {width: 1920.0, height: 1080.0}, visiblePosition: {x: 0.0, y: 0.0}, visibleSize: {width: 1920.0, height: 1040.0}
> name: \\.\DISPLAY4, size: {width: 3840.0, height: 1081.0}, visiblePosition: {x: 1920.0, y: 1.0}, visibleSize: {width: 1920.0, height: 1040.0}

At first, why second display has width 3840 px? It has 1920 px. 

Let's change displays' position
![изображение](https://user-images.githubusercontent.com/84628556/217024206-f0f85885-7b02-4620-928f-c705d65116b3.png)
Not code prints:
> name: \\.\DISPLAY1, size: {width: 1920.0, height: 1080.0}, visiblePosition: {x: 0.0, y: 0.0}, visibleSize: {width: 1920.0, height: 1040.0}
> name: \\.\DISPLAY4, size: {width: 0.0, height: 1080.0}, visiblePosition: {x: -1920.0, y: 0.0}, visibleSize: {width: 1920.0, height: 1040.0}

Second display now has width = 0 px. What??? How???

Next position:
![изображение](https://user-images.githubusercontent.com/84628556/217024681-d69fca9a-4fdb-424d-a62e-0c1eb3e99e09.png)
> name: \\.\DISPLAY1, size: {width: 1920.0, height: 1080.0}, visiblePosition: {x: 0.0, y: 0.0}, visibleSize: {width: 1920.0, height: 1040.0}
> name: \\.\DISPLAY4, size: {width: 1920.0, height: 2160.0}, visiblePosition: {x: 0.0, y: 1080.0}, visibleSize: {width: 1920.0, height: 1040.0}

Not second display has normal width, but height = 2160. :-(

Last position:
![изображение](https://user-images.githubusercontent.com/84628556/217025031-6981c1af-4436-4436-8554-22860247b15b.png)
> name: \\.\DISPLAY1, size: {width: 1920.0, height: 1080.0}, visiblePosition: {x: 0.0, y: 0.0}, visibleSize: {width: 1920.0, height: 1040.0},
> name: \\.\DISPLAY4, size: {width: 1920.0, height: 0.0}, visiblePosition: {x: 0.0, y: -1080.0}, visibleSize: {width: 1920.0, height: 1040.0},

At this time second display has height = 0. How it works???

N.b. Visible **positiion** all display in different positions: **work good**!
Visible size: height lost 40 pixels 

In conclusion: size of displays (if I have 2 and more displays) determines incorrect

**Comment by Roman-Dzerkal** [comment]
I found a way how to determine an actual screen size independent of position in screen settings!
At first we need to retrieve the data about size and position of a screen
```
int width = display.size.width.toInt();
int height = display.size.height.toInt();
int offsetX = display.visiblePosition!.dx.toInt();
int offsetY = display.visiblePosition!.dy.toInt(); 
```
where display - selected display


Actual parameters equal:
```
int actualWidth = width - offsetX;
int actualHeight = height - offsetY;
```

**Comment by lijy91** [comment]
Fixed, please update to 0.1.9
